unison-core/src/Unison/Reference.hs

```diff
@@ -180,6 +181,10 @@ groupByComponent refs = done $ foldl' insert Map.empty refs
       Map.unionWith (<>) m (Map.fromList [(Left r, [(k,r)])])
     done m = sortOn snd <$> toList m
 
+isBuiltin :: Reference -> Bool
+isBuiltin (Builtin _) = True
+isBuiltin _ = False
```

Comment:
There was a `isBuiltinType` somewhere else, but it didn't quite fit the need. Perhaps there is something somewhere that is better to use?